scoring.test.js

```diff
@@ -0,0 +1,236 @@
+const Scoring = require('./scoring');
+
+describe('Scoring a perfect game', () => {
+  it('reports total score', () => {
+    let mockScorecard = { pins: [
+      [10, 0],
```

Comment:
You could potentially DRY up your testing and remove a lot of these repeated mock ups: https://jestjs.io/docs/setup-teardown